manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MilosParipovic.OneCommander
+PackageVersion: 3.44.1
+PackageLocale: en-US
+Publisher: Milos Paripovic
+PublisherUrl: https://onecommander.com/
+PublisherSupportUrl: https://twitter.com/OneCommander
+Author: Milos Paripovic
+PackageName: OneCommander
+PackageUrl: https://onecommander.com/
+License: Copyright Milos Paripovic (©). All Rights Reserved
+Copyright: Copyright Milos Paripovic (©). All Rights Reserved
+CopyrightUrl: https://onecommander.com/
+ShortDescription: OneCommander is a modern file manager for Windows 10 and Windows 11.
+Description: OneCommander is a modern file manager for Windows 10 and Windows 11. Features include tabs, dual-pane browsers, columns navigation, built-in preview, a theming system, color tags, and much more. It is free for home use and has no ads.
+Moniker: OneCommander
+Tags:
+- OneCommander
+- Modern
+- File-Manager

```

```suggestion
Moniker: onecommander
Tags:
- onecommander
- modern
- file-manager
```
Best practice is to write moniker and tags in lower-case